package.json

```diff
@@ -67,6 +67,7 @@
     "karma-spec-reporter": "^0.0.36",
     "karma-webpack": "^5.0.0",
     "lerna": "^6.6.1",
+    "libp2p": "0.42.2",
```

Comment:
This removes the following warning: `warning " > @lodestar/reqresp@1.9.0" has unmet peer dependency "libp2p@~0.42.2".`

Not sure why this works but picked it up here: https://github.com/yarnpkg/yarn/issues/5347#issuecomment-1067264602